experiment/pod.yaml

```diff
@@ -0,0 +1,46 @@
+# Sample file for running e2e jobs in k8s
+# Steps:
+#   1. Prepare service-account.json and ssh-keys (priv, pub)
+#   2. Prepare a cluster (`gcloud container clusters create CLUSTER_NAME`)
+#   3. `kubectl create secret generic service-account --from-file=service-account.json=/path/to/service-account.json`
+#      `kubectl create secret generic ssh-key-secret --from-file=ssh-private=/path/to/rsa --from-file=ssh-public=/path/to/rsa.pub`
+#   4. Make sure your job sets `--mode=local` in config.json
+#   5. Replace JOB_NAME and --timeout in the yaml
+#   6. kubectl create -f pod.yaml
+#
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: k8s-e2e
+spec:
+  containers:
+  - name: k8s-e2e-test
+    image: gcr.io/k8s-testimages/kubekins-e2e-prow:0.3
+    args:
+    - "--timeout=60"
+    env:
+      - name: JOB_NAME
+        value: ci-kubernetes-e2e-prow-canary
```

Comment:
Note that it might interfere with the running job so might want to use a local project.

Most of this file is not needed for prow, only the [entry part](https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml#L274-L280) is needed, but yeah I'd be happy to make @ixdy happier next time (if he's got volunteered again)